src/types/address.d.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Addresses
+ * Description: The Addresses API allows you to organize customer addresses. Addresses are a sub-resource of a Customer.
+ * A Customer may have multiple addresses, such as home, work and neighbor. You are able to use a customer with an
+ * implicitBearer token. This is recommend for client-side interactions. Or you can use a client_credentials Bearer
+ * token. This is recommended for back end interactions.
+ * DOCS: https://documentation.elasticpath.com/commerce-cloud/docs/api/orders-and-customers/addresses/index.html
+ */
+import { core, Relationship } from './core'
+
+export as namespace address
+
+export namespace address {
```

Comment:
whats the difference between `export as namespace` and `export namespace`?

Reply:
a .d.ts files requires a export as namespace - not 100% sure why though. haha